### DISCUSSION TITLE
Failure to fetch image folder

### DISCUSSION THREAD

**Discussion by RedEarthRecreation** [community/community#49720] [Other Feature Feedback, Questions, & Ideas]
Anyone able to help me diagnose why my folder isn't being discovered by my Javascript.  I'm pretty new to all this but locally I can get things to work,  however on github the folder fails to be discovered.

**Comment by luigieterovik** [comment]
are you using Visual Studio Code?

**Comment by GhostOf0days** [comment]
JavaScript relative routing is from the page you are on (so HTML file) not actual relative routing like for HTML files.

**Reply by GhostOf0days** [reply]
This is when JavaScript is in a script tag for HTML.
